code/modules/vore/eating/belly_obj_vr.dm

```diff
@@ -237,7 +237,7 @@
 		START_PROCESSING(SSbellies, src)
 
 
-	create_reagents(100)	//CHOMP So we can have some liquids in bellies
+	create_reagents(custom_max_volume)	//CHOMP So we can have some liquids in bellies
```

Comment:
Might want to hardcode this too in order to avoid things becoming impossible to increase after decreasing it even once.